### PR TITLE
Counter badge on HomePage

### PR DESCRIPTION
**Pull request by niklashmbgr**

### CODE REVIEW

File: src/components/CartCounter/styled.js [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
 	margin: 1rem;
 	padding: 0.6rem;
 	color: whitesmoke;
 	font-weight: bold;
+	${({variant}) =>
+		variant === 'counterHome' &&
+		css`
+			display: inline-block;
+			margin: 0rem;
+			padding: 0rem;
```
Comment (JessicaLoers) [inline src/components/CartCounter/styled.js]: I'm not 100% sure, but I guess these properties have no effects.

Anyway if you set a value to  `0` there is no need to set a unit 🧐
```suggestion
			margin: 0;
			padding: 0;
```

File: src/components/Link/styled.js [hunk 2]
```diff
@@ -72,6 +72,13 @@ const StyledLink = styled.a`
 				color: black;
 				background-color: lightgray;
 		`}
+		${({variant}) =>
+		variant === 'counterHome' &&
+		css`
+			margin: 0rem;
+			padding: 0rem;
+			border: none;
```
Comment (JessicaLoers) [inline src/components/Link/styled.js]: Actually the same comment as above - I think you don't need the `border property` either.  
Test it in your dev tool. 
And please remove the unit 😬

File: src/components/Header/index.js [hunk 3]
```diff
@@ -2,10 +2,19 @@ import Link from 'next/link';
...
 import SVG from '../../../public/SVG/svg';
+import useStore from '../../hooks/useStore';
+import StyledCartCounter from '../CartCounter/styled';
+import StyledLink from '../Link/styled';
 
 import StyledHeader from './styled';
 
 export default function Header() {
+	const products = useStore(state => state.products);
+	const initialValue = 0;
```
Comment (JessicaLoers) [inline src/components/Header/index.js]: Just a nitty note - the name initialValue variable holds your initial quantity.  🤓
```suggestion
	const initialQuantity = 0;
```

File: src/components/CartCounter/styled.js [hunk 4]
```diff
@@ -1,8 +1,22 @@
...
+			width: 20px;
+			line-height: 20px;
+			color: whitesmoke;
+			font-size: small;
+			background-color: lightsteelblue;
+			border-radius: 50%;
+			text-align: center;
+		`}
```
Comment (JessicaLoers) [inline src/components/CartCounter/styled.js:20]: Another nitty comment 🙂

So far you have only used this styled component element only once. 
For this reason, one variant is perhaps a little overdosed for now.

**Review by JessicaLoers — COMMENTED** [review]
**Hi @niklashmbgr @LudiFlow** ✌️
well done!! 

Please have a look at my comments and please delete the units if the value is `0` 🤓

File: src/components/CartCounter/styled.js (hunk 4 above)
Comment (niklashmbgr) [reply src/components/CartCounter/styled.js:20]: actually this component is also used in the CartBanner/index.js so therefore i needed a variant...

File: src/components/Link/styled.js (hunk 2 above)
Comment (niklashmbgr) [reply src/components/Link/styled.js]: in this case: nobody has the intention to build a border :grimacing:

Comment (RobertBReich) [reply src/components/Link/styled.js]: > nobody has the intention to build a border

😂

**Comment by RobertBReich** [comment]
Looks like you covered and fixed all the things that were commented on. Also, a funny Honecker quote. 😄

**Pull request merged**